### PR TITLE
Fix Delete Modal Dialog, Standardise dialog, Admin delete button

### LINKED ISSUE #2282: Use common dialog section for all dialog components
**Actual Behaviour**

Currently present for some components

**Expected Behaviour**

Integrate with all components

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2282, #2293, #2308

Changes: 
- [x] Fix Delete Modal Dialog, make mobile responsive
- [x] Use common dialog section for all dialog components
- [x] Fix Admin Delete button not present in SkillPage

Demo Link: https://pr-2294-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
Screenshot 2019-06-19 at 2 54 20 AM

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd I need some help, for my commit 2. The Input field for update and delete of System Settings is not working(method `handleChange` changes not being reflected in input field). Please help

File: src/components/Admin/SystemSettings/DeleteSystemSettingsDialog.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+
+const DeleteSystemSettings = props => {
+  const { keyName, confirmDelete, handleClose } = props;
```
Comment (akshatnitd) [inline src/components/Admin/SystemSettings/DeleteSystemSettingsDialog.js]: Incorrect props. It should be `handleConfirm, handleClose, keyName`

File: src/components/Admin/SystemSettings/ChangeSystemSettingsDialog.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Button from '@material-ui/core/Button';
+
+const ChangeSystemSettings = props => {
```
Comment (akshatnitd) [inline src/components/Admin/SystemSettings/ChangeSystemSettingsDialog.js]: Rename it to UpdateSystemSettings

File: src/components/Admin/SystemSettings/ChangeSystemSettingsDialog.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  } = props;
+  return (
+    <React.Fragment>
+      <DialogTitle>{`${type} key`}</DialogTitle>
+      <DialogContent>
+        <TextField
+          label="Key Name"
+          placeholder="Key Name"
+          margin="normal"
+          value={keyName || ''}
```
Comment (akshatnitd) [inline src/components/Admin/SystemSettings/ChangeSystemSettingsDialog.js]: You should create state variables for this input fields in this component itself. If you pass, keyName from parent, it won't get passed to this, as the function call is only once.

Comment (fragm3) [reply src/components/Admin/SystemSettings/ChangeSystemSettingsDialog.js]: @akshatnitd If I localize this state to the inner component, the changes won't be shown in the parent component. How to handle that then?

**Comment by fragm3** [comment]
@akshatnitd The function of the update key is not yet present. I have fixed the delete key, but there seems some problem in the API(this PR sends the deleteKey correctly). Will fix it in my next PR.

**Comment by fragm3** [comment]
@akshatnitd made the changes, except delete key(server issue)

**Comment by fragm3** [comment]
I don't know why it's not working. I tested it 5 hours back, it worked. Also, the current susi.ai update key is not working as well. How about, I create a next PR and look into this? @akshatnitd

**Pull request merged**